scripts/data/src/org/akvo/gae/remoteapi/RemoteAPI.java

```diff
@@ -60,16 +76,21 @@ public static void main(String[] args) {
 
         final RemoteApiInstaller installer = new RemoteApiInstaller();
 
+        String orgId = args[1];
```

Comment:
rather than add this as an argument that has to be entered, how about we get it from the `appengine-web.xml` as well (like you do with the `eventNotification` property?

Reply:
There is the special case for local development, which does not require any files. See a further up the method how args[1] is used.

In any case, changed the method to make this more obvious. `orgId` is now used everywhere instead of `args[1]`